o2a/utils/el_utils.py

```diff
@@ -208,38 +114,65 @@ def comma_separated_string_to_list(line: str) -> Union[List[str], str]:
     return values[0] if len(values) <= 1 else values
 
 
+def _resolve_name_node(translation: str, props: PropertySet) -> Optional[str]:
+    """
+    Check if props include nameNode, nameNode1 ro nameNode2 value.
+    """
+    merged = props.merged
+    for key in ["nameNode", "nameNode1", "nameNode2"]:
+        start_str = "{{" + key + "}}"
+        name_node = merged.get(key)
+        if translation.startswith(start_str) and name_node:
+            return name_node
+    return None
```

Comment:
It's more of a question than complaint. Why do we need to treat nameNode separately and why we do the same with namNode1 and nameNode2? Those seem to be same parameters as all others?

Reply:
You are right. I have pushed an improvement before seeing your review. Discussing with @mik-laj we agreed to handle only `nameNode, nameNode1, nameNode2`.